emission/net/ext_service/habitica/sync_habitica.py

```diff
@@ -0,0 +1,68 @@
+# Standard imports
+import json
+import requests
+import logging
+import arrow
+import attrdict as ad
+
+# Our imports
+import emission.core.get_database as edb
+import emission.net.ext_service.habitica.proxy as proxy
+import emission.analysis.result.metrics.simple_metrics as earmts
+import emission.analysis.result.metrics.time_grouping as earmt
+
+
+
+def create_habit(user_id, new_habit):
+  method_uri = "/api/v3/tasks/user"
+  #how to pass in params? https://habitica.com/apidoc/#api-Task-GetUserTasks
+  #FIX! right now it's returning all types of tasks, I want it to return only "habits"
+  params = {'type': "habits"}
+  result = proxy.habiticaProxy(user_id, 'GET', method_uri, params)
+  habits = result.json()
+  for habit in habits['data']:
+    if habit['text'] == new_habit['text']:
+      return habit['_id']
+  #if habit not found, create habit
+  response = proxy.habiticaProxy(user_id, 'POST', method_uri, new_habit)
+  habit_created = response.json()
+  return habit_created['data']['_id']
+
+
+
+def reward_active_transportation(self, user_id):
+  #make sure habits exist
+  #bike
+  bike_habit = {'type': "habit", 'text': "Bike", 'up': True, 'down': False, 'priority': 2}
+  bike_habit_id = create_habit(user_id, bike_habit)
+  #walk
+  walk_habit = {'type': "habit", 'text': "Walk", 'up': True, 'down': False, 'priority': 2}
+  walk_habit_id = create_habit(user_id, walk_habit)
+
+  #get timestamps
+  user_val = list(edb.get_habitica_db().find({"user_id": user_id}))[0]
+  timestamp_from_db = user_val['last_timestamp']
+  timestamp_now = arrow.utcnow().timestamp
+  
+  #FIX! how do I change the frequency to be just one segment? For now I just left it as yearly
+  summary_ts = earmt.group_by_timestamp(user, timestamp_from_db, timestamp_now, earmt.LocalFreq.YEARLY, earmts.get_distance)
+
+  #FIX! is this getting the distances properly? Is it possible to have a function that calls "create sample data" to test the metrics
+  bike_distance = summary_ts[0].BICYCLING
+  walk_distance = summary_ts[0].ON_FOOT
```

Comment:
You can load sample data using the instructions in the README.
I can also give you ~ 6 mos of data from me, and you can use that to experiment with real data.